lib/secure-key-manager.ts

```
@@ -1,5 +1,10 @@
 import { Platform } from 'react-native';
-import * as SecureStore from 'expo-secure-store';
+// expo-secure-store might not be available in all environments; provide a soft import
+let SecureStore: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)